ngBoardFrontend/src/app/board/add-note-modal/add-note-modal.component.ts

```diff
@@ -13,7 +13,7 @@ export class AddNoteModalComponent {
 
   @ViewChild('modal') modal: ModalComponent;
 
-  @Output() noteCreated: EventEmitter<Note> = new EventEmitter();
+  @Output() noteCreated: EventEmitter<Note> = new EventEmitter<any>();
```

Comment:
new EventEmitter pa u ovm uglastim zagradama Note umesto any, to sam ja zaboravio da ubacim